clouddriver-kubernetes/src/test/java/com/netflix/spinnaker/clouddriver/kubernetes/caching/view/provider/KubernetesDataProviderIntegrationTest.java

```diff
@@ -517,6 +521,20 @@ void getArtifactsWrongAccount(SoftAssertions softly) {
     softly.assertThat(artifacts).isEmpty();
   }
 
+  @Test
+  void getClusterAndSortAscending(SoftAssertions softly) {
+    List<KubernetesManifest> manifests =
+        manifestProvider.getClusterAndSortAscending(
+            ACCOUNT_NAME, "backend-ns", "replicaSet", "replicaSet backend", Sort.AGE);
+    softly.assertThat(manifests).isNotNull();
```

Comment:
In this case I'd actually just do a regular `assertThat` because if `manifests` is `null` the next assertion will just NPE anyway.